package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-depcheck",
   "description": "Depcheck Grunt plugin",
-  "version": "0.0.1",
+  "version": "0.1.3",
```

Comment:
Not sure what had happened here so I have updated this to match version currently published to npm, which aside from package number seems to match

Reply:
The version is updated by travis on publish, we don't keep this one up to date